ftw/subsite/browser/contact.py

```diff
@@ -121,9 +110,33 @@ def send_feedback(self, recipient, subject, message, sender):
                 # if none is registered on the page in plone5
                 from_email_field._set_value('site@nohost.com')
                 email_from_email = from_email_field.value
-        # send the message
-        mh.send(msg, mto=email_from_email,
-                mfrom=[email_from_email])
+
+            email_from_email = (email_from_email or
+                                reg._records.get('plone.email_from_address').value)
+            email_from_name = (email_from_name or
+                               reg._records.get('plone.email_from_name').value)
+
+        # all the strings fed to send_mail have to be utf-8
```

Comment:
What about passing all the arguments of `send_mail()` through `safe_utf8()` in the method itself instead of converting it before passing it to the method? This way the programmer must noch remember to convert the data.

Reply:
Great Idea! I didn't think about it, narrow minded from a day looking at the same problem.